lua/telescope/actions/utils.lua

```diff
@@ -78,4 +78,26 @@ function utils.map_selections(prompt_bufnr, f)
   end
 end
 
+local findnth = function(str, nth)
+  local array = {}
+  for i in string.gmatch(str, "%d+") do
+    table.insert(array, tonumber(i))
+  end
+  return array[nth]
+end
+
+--- Utility to collect mappings of prompt buffer in array of `{mode, keybind, name}`.
+---@param prompt_bufnr number: The prompt bufnr
+function utils._get_registered_mappings(prompt_bufnr)
```

Comment:
Why prefix with `_`? I am fine with having this as stable api without prefix or doing two `_` prefix so it doesnt get exported with docgen